jbmc/src/java_bytecode/nondet.cpp

```diff
@@ -0,0 +1,125 @@
+/// Author: Diffblue Limited. All Rights Reserved.
+
+#include "nondet.h"
+
+#include <java_bytecode/java_types.h>
```

Comment:
I don't see where this would be used in this file, and, really, this should not live in the `java_bytecode` folder - it's way too big already.

Reply:
@tautschnig I see your point, but this is moved from TG to avoid duplicating code unneccesarily. Making this more general and non-Java specific is suggested by @peterschrammel too, but it should maybe  be a separate PR.

Reply:
It's not particularly clear to me what is actually Java-specific in this file? I thought it could be moved to a different folder with the single modification of dropping this (unnecessary) include?